src/gas/script-api.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const API_URL = "https://script.googleapis.com/v1/projects";
+
+class ScriptApi {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/fd132db2233355d7308db18ded515a069ff50e80/src/gas/script-api.js#L5"></span>‘ScriptApi’ is defined but never used.</td>
    </tr>
  </tbody>
</table>